modules/edu.gemini.seqexec.web/edu.gemini.seqexec.web.server/src/main/scala/edu/gemini/seqexec/web/server/http4s/SeqexecUIApiRoutes.scala

```diff
@@ -80,36 +80,44 @@ class SeqexecUIApiRoutes(auth: AuthenticationService, q: engine.EventQueue, se:
 
   def userInRequest(req: Request) = req.attributes.get(JwtAuthentication.authenticatedUser).flatten
 
-  val protectedServices: HttpService = tokenAuthService { GZip { HttpService {
-      case req @ GET -> Root / "seqexec" / "events"         =>
-        // Stream seqexec events to clients and a ping
-        val user = userInRequest(req)
-        WS(Exchange(pingProcess ++ Process.emit(Binary(trimmedArray(SeqexecConnectionOpenEvent(user)))) merge
-          se.eventProcess(q).map(v => Binary(newTrimmedArray(v))), scalaz.stream.Process.empty))
-
-      case req @ POST -> Root / "seqexec" / "logout"        =>
-        // Clean the auth cookie
-        val cookie = Cookie(auth.config.cookieName, "", path = "/".some,
-          secure = auth.config.useSSL, maxAge = Some(-1), httpOnly = true)
-        Ok("").removeCookie(cookie)
-
-      case req @ GET -> Root / "seqexec" / "sequence" / oid =>
-        val user = userInRequest(req)
-        user.fold(Unauthorized(Challenge("jwt", "seqexec"))) { _ =>
-          val r = for {
-            obsId <- \/.fromTryCatchNonFatal(new SPObservationID(oid)).leftMap((t:Throwable) => Unexpected(t.getMessage))
-            s     <- se.odbProxy.read(obsId)
-          } yield (obsId, s)
-
-          r match {
-            case \/-((i, s)) => se.load(q, i) *>
-                Ok(List(Sequence(i.stringValue(), SequenceState.NotRunning, "F2", s.toSequenceSteps, None)))
-            case -\/(e)      => NotFound(SeqexecFailure.explain(e))
-          }
-        }
-    }}
-
-  }
+  val protectedServices: HttpService =
+    tokenAuthService {
+      GZip {
+        HttpService {
+          case req @ GET -> Root / "seqexec" / "events"         =>
+            // Stream seqexec events to clients and a ping
+            val user = userInRequest(req)
+            WS(
+              Exchange(
+                Process.emit(Binary(trimmedArray(SeqexecConnectionOpenEvent(user)))) ++
```

Comment:
Yes, and some extra parentheses to get the associations of the operators right. The rest is just indentation, I should have mentioned it when submitting the PR.